src/template-analyzer.ts

```diff
@@ -198,19 +198,39 @@ export class TemplateAnalyzer {
    */
   public resolveLocation(
     loc: parse5.Location
-  ): ESTree.SourceLocation | null | undefined {
+  ): ESTree.SourceLocation {
     let offset = 0;
+    let height = 0;
 
     for (const quasi of this._node.quasi.quasis) {
```

Comment:
above this, check if `this._node.loc` or `this._node.quasi.loc` is empty and, if it is, return `null`